grafana/provider_test.go

```diff
@@ -248,6 +223,16 @@ func accTestsEnabled(t *testing.T, envVarName string) bool {
 	return enabled
 }
 
+func checkEnvVarsSet(t *testing.T, envVars []string) {
```

Comment:
We could make this accept variadic arguments instead, and then you just call it like `checkEnvVarsSet(t, "FOO", "BAR", "QUUX")`
```suggestion
func checkEnvVarsSet(t *testing.T, envVars ...string) {
```

WDYT?

Reply:
Yes, that's much cleaner, thanks